app/models.py

```diff
@@ -0,0 +1,21 @@
+from sqlalchemy import orm
+from sqlalchemy.dialects.postgresql import JSON
+from flask_sqlalchemy import SQLAlchemy, sqlalchemy.orm
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship, backref
+import sqlamp
+
+Base = declarative_base()
+"""in this db i have 3 columns name id and categories_path"""
+class categories(Base):
```

Comment:
According to PEP8 (https://www.python.org/dev/peps/pep-0008/) class names should start with upper case